### PR TITLE
fix typos and error texts

### PR DESCRIPTION
**Pull request by nasermirzaei89**
In this PR I fixed these issues:
* Error string should not be capitalized or end with punctuation. (This convention is the concern of wrapping errors)
* Remove redundant types
* Fix typos

_This PR won't fix any critical issues._

### CODE REVIEW

File: query_test.go [hunk 1]
```diff
@@ -138,18 +138,18 @@ func TestInsertSliceStructPtr(t *testing.T) {
 	now = time.Now()
 
 	data := []*testRow{
-		&testRow{
+		{
```
Comment (martinrode) [inline query_test.go:141]: Cool, didnt know you can do that!

File: exec.go [hunk 2]
```diff
@@ -142,7 +142,7 @@ func (db *DB) InsertBulk(table string, data interface{}) error {
 	}
 
 	if structMode {
-		return fmt.Errorf("InsertBulk: Need Slice to insert bulk.")
+		return fmt.Errorf("need slice to insert bulk")
```
Comment (martinrode) [inline exec.go:145]: I prefer starting with the function name, so "InsertBulk: need slice".

File: exec.go [hunk 3]
```diff
@@ -114,7 +114,7 @@ func setPrimaryKey(rv reflect.Value, id int64) {
 	case reflect.Uint64:
 		rv.SetUint(uint64(id))
 	default:
-		err := fmt.Errorf("Unknown type to set primary key: %s", rv.Type())
+		err := fmt.Errorf("unknown type to set primary key: %s", rv.Type())
```
Comment (martinrode) [inline exec.go:117]: I that a rule (good practise) I dont know about? Starting error messages with small case?

**Review by martinrode — COMMENTED** [review]
Naser, thanks for these changes, pls see my comments.

File: exec.go (hunk 3 above)
Comment (nasermirzaei89) [reply exec.go:117]: Yep

File: exec.go (hunk 2 above)
Comment (nasermirzaei89) [reply exec.go:145]: It's a refactor with that practice (don't start with uppercase and end with punctuation).
Also, You mentioned the function name in the sentence: need slice to **insert bulk**

File: query_test.go (hunk 1 above)
Comment (nasermirzaei89) [reply query_test.go:141]: I suggest you use modern IDEs and add lint tests (https://golangci.com/) to your codes.

**Pull request closed without merging**